_data/standing_meetings.yml

```diff
@@ -5,6 +5,10 @@
   agenda:
     - title: Updates
       link: "[Meeting Notes](https://docs.google.com/document/d/1HMUtM8rRPeTYDqsWZVD-iQityjJj9_GKgu0FHHbiBr0)"
+    - title: "Add support for differentiating with respect to user-defined types -- Progress Update"
```

Comment:
Can you move this as a last entry. We do most recent first.

Reply:
Done.